Examples/Framework/include/ActsExamples/EventData/Measurement.hpp

```diff
@@ -117,6 +119,12 @@ class MeasurementContainer {
     return getMeasurement<Size>(addMeasurement(Size));
   }
 
+  template <typename... Args>
+  VariableProxy emplaceVariableMeasurement(std::uint8_t size, Args&&... args);
+
+  template <std::size_t Size, typename... Args>
+  FixedProxy<Size> emplaceFixedMeasurement(Args&&... args);
```

Comment:
If we use concrete parameter types here we could overload an `emplaceFixedMeasurement` function for both cases. Nothing critical but that might align more with the rest of the API

Reply:
What I mean is that there could be a single function called `emplace` or `emplaceMeasurement` and one has the overload with the size template for fixed size proxy returns. But I think that only works if you type out the parameters otherwise C++ will not let us overload it.

Reply:
Ah right! Let me see if I can get that to work.

Reply:
Wow, that went surprisingly easily.